plugins/axelix-maven-plugin/src/integrationTest/copy-report/target/spring-test-profiler/latest.html

```
@@ -0,0 +1 @@
+<h1>test</h1>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)